src/main/java/net/dv8tion/jda/core/entities/User.java

```diff
@@ -109,6 +109,13 @@
      */
     String getEffectiveAvatarUrl();
 
+    /**
+     * The "tag" for this user, this is a shortcut to {@link java.lang.String#format(String, Object...) String#format("%#s", user)}
+     *
+     * @return The tag for this user
+     */
+    String getAsTag();
```

Comment:
Add an example here for better understanding of what this means

Reply:
I'm not sure if we should also clarify that this is not for mentioning and that IMentionable#getAsMention() should be used for that instead.

Also just FYI: the String#format() link doesn't render i think, since that method is not part of JDA and we are not rendering docs for java itself (so they are not linked). It isn't really an error (it will just render to blank text without any linking/jdoc syntax), but i guess it could be removed as it doesn't really do a lot.

Reply:
The link doesn't render in javadocs indeed, but it does in ide's (tested in intellij) so it might be useful to leave it there
![javadocs](https://i.entered.space/UDLDlGB.png)
![ide](https://im.going-g.host/Wc8HSa5.png)

Reply:
We use the call operator consistently in the docs, so the proper alias would be `String.format(...)`